client/state/action-types.js

```diff
@@ -288,6 +288,10 @@ export const LAYOUT_NEXT_FOCUS_SET = 'LAYOUT_NEXT_FOCUS_SET';
 export const LOGIN_REQUEST = 'LOGIN_REQUEST';
 export const LOGIN_REQUEST_FAILURE = 'LOGIN_REQUEST_FAILURE';
 export const LOGIN_REQUEST_SUCCESS = 'LOGIN_REQUEST_SUCCESS';
+export const LOGIN_TWOFACTOR_UPDATE_NONCE = 'LOGIN_TWOFACTOR_UPDATE_NONCE';
+export const LOGIN_TWOFACTOR_PUSH_POLL_COMPLETED = 'LOGIN_TWOFACTOR_PUSH_POLL_COMPLETED';
+export const LOGIN_TWOFACTOR_PUSH_POLL_START = 'LOGIN_TWOFACTOR_PUSH_POLL_START';
+export const LOGIN_TWOFACTOR_PUSH_POLL_STOP = 'LOGIN_TWOFACTOR_PUSH_POLL_STOP';
```

Comment:
The recent action types that were added for two step authentication are `TWO_FACTOR_AUTHENTICATION_LOGIN_*` and `TWO_FACTOR_AUTHENTICATION_SEND_SMS_CODE_REQUEST_*`. That's not in line with those brand new ones. We should make them consistent (either by renaming the former ones, or the new ones).